src/main/java/org/fundacionjala/coding/dennis/CesarCryp.java

```diff
@@ -0,0 +1,58 @@
+package org.fundacionjala.coding.dennis;
+
+/**
+ *
+ */
+public class CesarCryp extends Encrypto {
+
+    public static final int FIRSTPOS = 0;
+    public static final int THEREISNO = -1;
+
+    /**
+     *
+     */
+    public CesarCryp() {
+
+    }
+
+    /**
+     *
+     * @param key llave.
+     * @param mess mensage.
+     * @return mensaje encriptado.
+     */
+    @Override
+    public String crypto(final String key, final String mess) {
+        int keyNum = Integer.parseInt(key);
+        StringBuilder messCryp = new StringBuilder(1);
+        for (char letter : mess.toCharArray()) {
+            int pos = alphaS.indexOf(letter);
+            letter = pos == THEREISNO ? letter : getLetter(pos, keyNum);
+            messCryp.append(letter);
+        }
+        return messCryp.toString();
+    }
+
+    /**
+     *
+     * @param key llave.
+     * @param encMess mensaje encriptado.
+     * @return mensaje desencriptado.
+     */
+    @Override
+    public String decrypto(final String key, final String encMess) {
+        return crypto(Integer.toString(Integer.parseInt(key) * THEREISNO), encMess);
+    }
+
+    /**
+     *
+     * @param pos posicion.
+     * @param key llave.
+     * @return reemplazo.
+     */
+    private char getLetter(final int pos, final int key) {
```

Comment:
It looks like both implementations have getLetter method.
May this method be abstract ?